src/heap.js

```diff
@@ -1,41 +1,86 @@
 /* eslint-disable */
 class Heap {
-    constructor() {
-        this.storage = [null];
-        this.size = 0;
-    }
-    // Inserts the given value in the heap
-    // Calls bubbleUp in order to put the newly-inserted element in the right place in the heap
-    insert(val) {
-        
-    }
-    // Deletes the element located at the last index of the heap
-    // Calls siftDown in order to put the newly-inserted element in the right place in the heap
-    delete() {
-
-    }
-    // Returns the maximum value in the heap in constant time
-    getMax() {
-
-    }
-    // Returns the size of the heap
-    getSize() {
-
-    }
-    // Returns the storage array
-    getStorage() {
-
-    }
-    // Moves the element at the specified index "up"
-    // the heap such that the heap property is maintained
-    bubbleUp(index) {
-
-    }
-    // Moves the element at the specified index "down"
-    // the heap such that the heap property is maintained
-    siftDown(index) {
-
-    }
+  constructor() {
+      this.storage = [null];
+      this.size = 0;
+  }
+  // Inserts the given value in the heap
+  // Calls bubbleUp in order to put the newly-inserted element in the right place in the heap
+  insert(val) {
+    const heap = this.getStorage();
```

Comment:
Try to keep the formatting and indentation here consistent with the rest of your code. It's very jarring to read improperly-formatted code.